tools/performance/benchmarks-data/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "benchmarks-data",
+  "version": "0.0.0-dev",
+  "description": "Utilities for fetching and storing benchmark data from GH Actions.",
+  "author": {
+    "name": "Enso Team",
+    "email": "contact@enso.org"
+  },
+  "homepage": "https://github.com/enso-org/enso",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:enso-org/enso.git"
+  },
+  "main": "target/index.js",
+  "type": "module",
+  "scripts": {
+    "start": "vite-node src/index.ts",
+    "build": "tsc",
+    "test": "vitest",
+    "lint": "eslint src",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "commander": "^12.1.0",
+    "date-fns": "^3.6.0",
+    "jszip": "^3.10.1",
+    "octokit": "^4.0.2",
+    "vite": "^5.2.13",
```

Comment:
You are using `vite-node` in the start script, but depending on `vite`. This should be a `vite-node` dependency instead.